CHANGELOG.md

```diff
@@ -1,66 +1,46 @@
 ## 3.18.0 (Unreleased)
```

Comment:
looks like this file got accidentally committed? can you remove?

Reply:
Yes, removed it